ejb3/src/main/java/org/jboss/as/ejb3/logging/EjbLogger.java

```diff
@@ -3204,4 +3204,8 @@
     @LogMessage(level = WARN)
     @Message(id = 515, value = "[EJB3.2 spec, section 4.9.2] Singleton session beans are not allowed to implement 'javax.ejb.SessionBean' interface. This interface on bean '%s' is going to be ignored and should be removed.")
     void singletonCantImplementSessionBean(String className);
+
+    @LogMessage(level = WARN)
+    @Message(id = 516, value = "Potential Interface view not exposed... : %s")
```

Comment:
I'm not sure if WARN is appropriate here as this might be emitted due to a legitimate design choice by the application author.

I think if we log a WARN or above for something that might be ok with the user it should be possible to turn off the log category without losing other important information. That would mean a separate log category. That seems like overkill for this though. Perhaps INFO suffices, since my impression is the goal here is to help users understand why something doesn't work as they expect. If something isn't working reading the INFO level logging emitted as the app is deployed is a reasonable thing to expect.

Reply:
Resolved.